### ISSUE TITLE
Pure component transform transforms classes with class properties initializers

### ISSUE THREAD

**Issue by okonet** [reactjs/react-codemod#85]
This class that is using class property initializers

``` jsx
import React, { PropTypes } from 'react';

export default class StringControl extends React.Component {
  handleChange = (e) => {
    this.props.onChange(e.target.value);
  };

  render() {
    return <input type="text" value={this.props.value || ''} onChange={this.handleChange} />;
  }
}

StringControl.propTypes = {
  onChange: PropTypes.func.isRequired,
  value: PropTypes.node,
};
```

gets transformed to:

``` jsx
import React, { PropTypes } from 'react';

export default function StringControl(props) {
  return <input type="text" value={props.value || ''} onChange={this.handleChange} />;
}

StringControl.propTypes = {
  onChange: PropTypes.func.isRequired,
  value: PropTypes.node,
};
```

It should not.

**Comment by cpojer** [comment]
Hey Andrey!

We don't actually have resources to fix bugs in this repo, we simply share what works for us or other members in the community. If you'd like see this fixed you'll likely have to send a pull request yourself :)

**Comment by danharper** [comment]
I've got a PR open to fix this :) #84

**Comment by okonet** [comment]
@cpojer I think we have a winner here! Can the #84 from @danharper be merged any soon?
